index.html

```diff
@@ -0,0 +1,264 @@
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <!-- HEADER SECTION -->
+    <header class="header">
+      <div class="header-links">
+        <div class="header-left"></div>
+        <div class="header-right">
+          <ul class="header-link">
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fab fa-facebook"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fab fa-twitter"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fa-brands fa-linkedin-in"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fa-brands fa-instagram"></i>
+              </a>
+            </li>
+          </ul>
+          <div class="header-items">
+            <a href="#">English</a>
+            <a href="#">Logout</a>
+          </div>
+        </div>
+      </div>
+      <nav>
+        <div class="nav-center">
+          <div class="nav-header">
+            <div class="logo">
+              <a href="#">
+                <img
+                  src="./assets/Images/conference-logo.png"
+                  alt="conference"
+                />
+              </a>
+            </div>
+            <div class="nav-toggle">
+              <i class="fa-solid fa-bars"></i>
+              <i class="fa-solid fa-times"></i>
+            </div>
+          </div>
+          <ul class="links">
+            <li class="home"><a href="#about">Home</a></li>
+            <li class="projects"><a href="about.html">About</a></li>
+            <li class="about"><a href="#join">Join</a></li>
+          </ul>
+        </div>
+      </nav>
+      <section class="headline-container">
+        <div class="headline">
+          <div class="hero-header-title">
+            <h3>"Hello! Sharing World"</h3>
+          </div>
+          <div class="hero-title">
+            <h1>International Conference on Sustainable Water Management</h1>
+          </div>
+          <div class="hero-desc">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaerat
+              culpa illum et neque veritatis, non itaque quasi deleniti nostrum
+              nesciunt. Quaerat culpa illum et neque veritatis, non itaque quasi
+              deleniti nostrum nesciunt.
+            </p>
+          </div>
+          <div class="hero-text-date">
+            <h2>24th - 25th Nov 2022</h2>
+          </div>
+          <div class="hero-bottom">
+            <p>
+              International Conference on Sustainable Water Management (ICSWM)
+            </p>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] On screen sizes between 480px and 768px your top section has a lot of whitespace on top. Kindly reduce that margin to remove this whitespace and make your project look like the template you are following.
 - Your design:
 - 
<img width="658" alt="image" src="https://user-images.githubusercontent.com/61919617/203631404-c6fa2987-7848-4931-a6c9-ee4557318d30.png">

- Template: 
- 
<img width="402" alt="image" src="https://user-images.githubusercontent.com/61919617/203631468-fb7254ec-9341-45ac-a835-765831d53985.png">
<img width="402" alt="image" src="https://user-images.githubusercontent.com/61919617/203631467-9f1b4ffc-5b61-4483-8112-26e516e7c407.png">